src/main/java/net/sf/jabref/gui/entryeditor/PdfCommentsTab.java

```diff
@@ -66,8 +66,9 @@ private void setUpPdfCommentsTab() {
             DefaultListModel<String> listModel = new DefaultListModel<>();
             commentList.setModel(listModel);
             PdfCommentImporter commentImporter = new PdfCommentImporter();
-            HashMap<String, String> importedNotes = commentImporter.importNotes(field.get());
-            for (String note : importedNotes.values()) listModel.addElement(note);
+            String pdfPath = field.get().replaceFirst(".*?:", System.getProperty("file.separator")).replaceAll(":PDF", "");
```

Comment:
Just a hint, there is a convenience method in `FileUtil `which does the job for you and because the file field could contain more than one file entry.
getListOfLinkedFiles(...) just look at the call hierarchy of that method to see how to use it...